CMakeLists.txt

```diff
@@ -73,8 +73,11 @@ endif()
 
 option(WITH_GMX "Build gromacs reader/writer, disabling leads to reduced functionality!" ON)
 if (WITH_GMX)
-  find_package(GROMACS 20160000 REQUIRED)
-  if(DEFINED GROMACS_VERSION AND GROMACS_VERSION VERSION_GREATER_EQUAL "20200000")
+  find_package(GROMACS 2021 QUIET CONFIG NAMES gromacs gromacs_d)
+  if(NOT GROMACS_FOUND)
+    find_package(GROMACS 2016 REQUIRED MODULE)
```

Comment:
How come you don't need the four 0000 in this case?

Reply:
We converted the version to match with upstream, 20160001 is now 2016.1.

Reply:
I see.